src/structures/channel/voice/VoiceChannel.ts

```diff
@@ -0,0 +1,3 @@
+import { VoiceBasedChannel } from './VoiceBasedChannel';
+
+export class VoiceChannel extends VoiceBasedChannel {}
```

Comment:
🤔

Reply:
This structure is there for the future methods that will be inside.